osba.sh/index.html

```diff
@@ -76,25 +76,25 @@ <h3>Azure SQL Database</h3>
 
     <div class="row" style="padding-top: 2em;">
       <div class="small-12 columns">
-        <h2>Experimental Services</h2>
+        <h2>Services Coming Soon</h2>
       </div>
     </div>
 
     <div class="row">
       <div class="small-4 columns">
-        <a href="https://github.com/Azure/open-service-broker-azure/blob/master/docs/modules/cosmosdb.md">
+        <a href="">
```

Comment:
Instead of blank links, can we link to anything that explains what coming soon means, where they went, and when they are coming back? Maybe we could link to a draft of the GA release notes temporarily until we ship?

Reply:
Maybe we should link to the roadmap and add some text about that there (and make sure the roadmap indicates they aren't available now)? I don't think we know when they are coming back, so it's probably more confusing to try and answer that? We won't have them after Kent's upcoming PR and they won't be something you can turn on until we add them back in (they are going to be excluded from compilation).

Reply:
Yeah the roadmap would work great! 👍